src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc

```diff
@@ -0,0 +1,34 @@
+[id="adding-custom-certificates-to-trust-store_{context}"]
+= Adding custom public SSL certificates to {prod-short} trust-store
+
+This procedure describes how to configure {prod-short} to be able to handle HTTP requests from untrusted sources.
+
+.Procedure
+. Save the certificate that you want to apply to `ca.crt` file.
+. Create a new *configMap* with the certificate:
++
+[subs="+quotes"]
+----
+$ kubectl create configmap custom-public-cert --from-file=ca.crt -n=che
+----
+. Set the `global.customPublicCert` property to `true`.
++
```

Comment:
There is nothing about this property in lines below.
How we set it to `true` ?

Reply:
The instruction is below

Reply:
fixed the title